src/main/res/values/strings.xml

```diff
@@ -652,6 +652,9 @@
     <string name="accept_invalid_certificates">Accept invalid certificates</string>
     <string name="switch_account">Switch Profile</string>
     <string name="add_account">Add Profile</string>
+    <string name="profile_tag">Private Tag</string>
+    <string name="profile_tag_hint">eg. Work, Family, ...</string>
```

Comment:
i think, either `eg.` or `...` is enough:

```suggestion
    <string name="profile_tag_hint">eg. Work, Family</string>
```

or 

```suggestion
    <string name="profile_tag_hint">Work, Family, …</string>
```

in any case, `...` which should be an ellipsis, so `…` and not `...`